hubble-fe/src/components/common/AppBar.tsx

```diff
@@ -1,19 +1,31 @@
-import React, { useCallback } from 'react';
+import React, { useCallback, useState } from 'react';
 import { observer } from 'mobx-react';
 import { useLocation } from 'wouter';
-
+import { useTranslation } from 'react-i18next'
+import { Select } from 'antd';
 import './AppBar.less';
+const { Option } = Select;
 
 const AppBar: React.FC = observer(() => {
   const [_, setLocation] = useLocation();
-
+  // init select language
+  const [languageType, setLanguageType] = useState(localStorage.getItem('languageType') || 'zh-CN')
+  const { t } = useTranslation()
   const setRoute = useCallback(
     (route: string) => () => {
       setLocation(route);
     },
     [setLocation]
   );
-
+  /*
+  * switch language
```

Comment:
expect to align with the previous "*" and capitalize the first letter "switch language"